### ISSUE TITLE
Questions about SLUE challenge 2023

### ISSUE THREAD

**Issue by GT-KIM** [asappresearch/slue-toolkit#32]
Hello,
I have two questions about the challenge rule.

1. What is the difference between pipeline and end2end? In my understanding, the "pipeline" model has two neural network models independently: audio-to-text (ASR) and text-to-entity (NLP). The "E2E" model has one neural network model: audio-to-(entity, word). 
If my model can estimate the whole transcript and entity of each word in only one neural network, is it "E2E"?
Or, if my model has multiple training steps(pretraining-finetuning-end2end) but has one inference step(end2end), it can be "E2E"?
I think if there is no "token id to string" stage in the inference step, it can be "E2E", but I'm not sure.

2. Where can I submit my submissions for participating challenge? Moreover, can we see the leaderboard or test results before the deadline?

Thanks

**Comment by sshon-asapp** [comment]
Conventional pipeline term in spoken language understanding indicates the system first generate transcription using ASR and then following NLP model generate final output. Thus, the each module trained with different objectives. End-to-end system means you have single model that trained with the single objective (e.g. NER in this case). As long as you train model in end-to-end manner (audio or audio feature to label directly), we can regard as it's end to end model. Pre-training+fine-tuning can be regarded as end-to-end system. 

Submission link will be updated soon in this page.
https://asappresearch.github.io/slue-toolkit/interspeech2023_submission.html

**Comment by GT-KIM** [comment]
I understand. Thank you!
